deepcomp/env/entities/user.py

```diff
@@ -29,7 +29,7 @@ def __init__(self, id, map, pos_x, pos_y, movement, dr_req=1):
         self.movement = movement
         self.dr_req = dr_req
         # dict of connected BS: BS (only connected BS are keys!) --> data rate of connection
-        self.bs_dr = dict()
+        self.bs_dr = {}
```

Comment:
Why is initializing with `{}` better than with `dict()`? I always feel like `dict()` is more explicit. With `{}`, I sometimes get confused whether it initializes a dict or a set.

Reply:
Fixing this improves the minor performance. Please have a look [here](https://deepsource.io/gh/withshubh/DeepCoMP/issue/PTC-W0019/description)